deployment/v3/docs/change-domain-name-in-a-running-env.md

```diff
@@ -0,0 +1,96 @@
+# Change domain name in a running env ( v2 )
+
+Follow the below steps to rename domain name in a running v2 env:
+1. Update the domain name in groups/all.yml
+2. Ran playbooks/nginx.yml
+3. Redeploy config-server, keycloak
+4. Update  the domain name in the  valid redirect URL & web origins of mosip keycloak-clients 
+5. Updated domain name admin-ui, prereg-ui configmaps. 
+6. Restart mosip modules.
+
+# Change domain name in a running env ( v3 )
+
+Follow the below steps to rename domain name in a running v3 env:
+1. Update DNS records from the old DNS name to the new DNS name i.e., (from `*.country-data.mosip.net` to `*.data.mosip.net` ).
+2. Generate SSL certs for the new domain.
+   * Backup the old SSL certs via the below command:
+     ```
+     sudo mv /etc/letsencrypt/live /etc/letsencrypt/live-{date}
+     ```
+   * Generate SSL certs for new domain via the below command:
+     ```
+     sudo certbot certonly --agree-tos --manual --preferred-challenges=dns -d *.<new-dns-name> -d <new-dns-name>
+     ```
+     Example:
+     ```
+     sudo certbot certonly --agree-tos --manual --preferred-challenges=dns -d *.data.mosip.net -d data.mosip.net
+     ```
+3. Update ssl certs location and domain names in nginx config file `/etc/nginx/nginx.conf` and restart nginx service.
+   ```
+   ssl_certificate /etc/letsencrypt/live/data.mosip.net/fullchain.pem;   
+   ssl_certificate_key /etc/letsencrypt/live/data.mosip.net/privkey.pem;
+   ```
+   ```
+   ## public IP server section
+   server{
+     listen 216.48.176.35:443 ssl;
+     server_name  api.data.mosip.net prereg.data.mosip.net resident.data.mosip.net esignet.data.mosip.net;
+     location / {
+        proxy_pass                      <http://myPublicIngressUpstream;>
+        proxy_http_version              1.1;
+        ....
+        ....
+       }
+   }
+   ```
+   ```
+   sudo systemctl restart nginx
+   ```
+4. Update the domain in `global` configmaps on the `default` namespace.
+5. Update the iam domain in `keycloak-host` configmaps on `keycloak` namespace.
```

Comment:
Shouldn't it be IAM? Please check..

Reply:
It is a keycloak namespace

IAM is only used for keycloak.